firebase.json

```diff
@@ -1,30 +1,18 @@
 {
```

Comment:
The changes in this file will break the scripts in `package.json`. Please rollback this change. If you need help using Firebase Hosting on your local environment, ask us how to use it.